include/retdec/pelib/ImportDirectory.h

```diff
@@ -555,6 +555,31 @@ namespace PeLib
 		m_ordinalMask = (uint64_t)1 << ((pointerSize * 8) - 1);
 	}
 
+	inline bool isBadImportName(const std::string & importName)
+	{
+		unsigned char theFirstChar;
+
+		// The name must have some characters
+		if(importName.size() == 0)
+			return true;
+		
+		// The first character of the name must not be an invalid ASCII char
+		theFirstChar = importName[0];
+		if(theFirstChar <= 0x20 || theFirstChar >= 0x80)
```

Comment:
Is there a reason for supporting the `0x7f` character (DEL) in import names? To put it into other words, shouldn't the second number in the condition be `0x7f` instead of `0x80`?